Sources/Fisticuffs/Subscribable.swift

```diff
@@ -37,9 +37,11 @@ open class Subscribable<Value> : AnySubscribable {
     open func subscribe(_ options: SubscriptionOptions, block: @escaping (Value?, Value) -> Void) -> Disposable {
```

Comment:
Just realized, should we also have a scheduler for `bind` methods?

Reply:
For the `bind` methods, I wonder if it should be hardcoded to `DispatchQueue.main` 🤔   since they're all for UIKit if I remember correctly

Reply:
I have mixed feelings about hardcoding to the main queue because `BindableProperty`s aren't restricted to just UI components, although that's what we've provided so far. On the other hand this makes things easier to manage for the users of bindings

Reply:
I think it makes sense to make the UIKit ones default to the main queue

Reply:
Ok I'll update it, noticed we were doing it internally in `Computed` too

Reply:
It might make sense to use something like this actually.

```swift
public struct ImmediateIfOnMainThread: Scheduler {
   public func schedule(_ action: @escaping () -> Void) {
        if Thread.isMainThread {
          action()
        }
        else {
          DispatchQueue.async(action)
        }
    }
}
```

I'm not sure if doing multiple `Dispatch.async` to update bindings (once for each updated binding) will cause UIKit to relayout & redraw many times in short bursts (if multiple bindings are updated at once)

Reply:
Ah I actually have that as extension in TSB lol, I can add it here too.

Reply:
Updated. Hope I did it right, will add tests shortly.